RoboFile.php

```diff
@@ -550,6 +550,22 @@ public function publish($opts = ['force' => false]) {
       ->run();
   }
 
+  function changelogUpdate($opts = ['version-name' => null]) {
+    $this->say("Updating changelog");
+    $outputs = $this->getChangelogController()->update($opts['version-name']);
+    if($opts['quiet']) {
+      return;
+    }
+    $this->say("Changelog \n{$outputs[0]} \n{$outputs[1]}\n\n");
+    $this->say("IMPORTANT NOTES \n" . ($outputs[2] ?: 'none'));
+  }
+
+  function changelogGet($opts = ['version-name' => null]) {
+    $outputs = $this->getChangelogController()->get($opts['version-name']);
+    $this->say("Changelog \n{$outputs[0]} \n{$outputs[1]}\n");
+    $this->say("IMPORTANT NOTES \n" . ($outputs[2] ?: 'none'));
```

Comment:
What about not printing it at all when it's empty?

Reply:
I think it is useful to print it to be assured that there are no important notes.